frame/evm/src/lib.rs

```diff
@@ -322,6 +322,26 @@ pub mod pallet {
 				pays_fee: Pays::No,
 			})
 		}
+
+		#[pallet::weight(0)] // TODO: define a weight
+		pub fn hotfix_inc_account_sufficients(
+			origin: OriginFor<T>,
+			addresses: Vec<H160>,
+		) -> DispatchResultWithPostInfo {
+			frame_system::ensure_signed(origin)?;
+			for address in addresses {
```

Comment:
Here we might want to check `addresses.len`. Regarding the weight, you can assume approx `T::DbWeight::get().reads_writes(addresses.len as Weight, addresses.len as Weight)` + the overweight we want to artificially add.